lms/djangoapps/certificates/api.py

```diff
@@ -549,8 +549,9 @@ def get_certificate_template(course_key, mode, language):
     Retrieves the custom certificate template based on course_key, mode, and language.
     """
     template = None
-    # fetch organization of the course
-    org_id = get_course_organization_id(course_key)
+    # fetch organization of the course, if there is one.
+    org = get_course_organization(course_key)
+    org_id = org["id"] if org else None
```

Comment:
The additional null check here is just replicating what the `util.organizations_helpers.get_course_organization_id` wrapper function did before I removed it.